exec/java-exec/src/main/resources/rest/static/js/ace-code-editor/snippets/sql.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Drill SQL Syntax Snippets
+ */
+
+ace.define("ace/snippets/sql",["require","exports","module"], function(require, exports, module) {
```

Comment:
1. Could you please add information about snippets to Jira description. We might what to add it to Drill documentation later so user knows which snippets can be used. Let's say, it's hard to guess the snippers without looking into the code :)

2. When using snippets, I saw the following warning.

![image](https://user-images.githubusercontent.com/15086720/34730558-6a34fde4-f568-11e7-81e9-3e6d62091189.png)
Could you please check?

Reply:
+1
I did see this initially, but ignored it as I made progress. Guess I got trained to ignoring the contents in the end :)
Will fix this.